app/org/sagebionetworks/bridge/play/controllers/AuthenticationController.java

```diff
@@ -52,8 +51,13 @@ public Result emailSignIn() {
         verifySupportedVersionOrThrowException(study);
         
         CriteriaContext context = getCriteriaContext(study.getStudyIdentifier());
-        
-        UserSession session = authenticationService.emailSignIn(context, signInRequest);
+        UserSession session = null;
+        try {
+            session = authenticationService.emailSignIn(context, signInRequest);
+        } catch(ConsentRequiredException e) {
+            logAuthenticationSuccess(e.getUserSession());
```

Comment:
log authentication success on failed sign in?  method name seems a little misleading.

Reply:
We're logging the session even though in this case, it will be returned with a 412 status code and not a 200 status code... changed method name to setCookieAndRecordMetrics.